src/actions/API.actions.js

```diff
@@ -95,6 +96,8 @@ export function createSUSIMessage(createdMessage, currentThreadID, voice) {
       Actions.sendToHardwareDevice(response);
 
       receivedMessage.text = response.answers[0].actions[0].expression;
+      // Setting Language received from User
+      receivedMessage.lang = response.answers[0].actions[0].language;
```

Comment:
I think the query model has this by default. If you check, all responses from server have this attribute compulsorily as @Orbiter mentioned in https://github.com/fossasia/susi_android/issues/853
Otherwise following best practices, we even need to have a check for if the `response.answers[0].actions[0]` contains `type` and `expression` which would be redundant because this is the basic query model being followed on the server.

http://api.susi.ai/susi/chat.json?q=hello